src/components/actions.js

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Ten plik nie powinien mieścić się w katalogu `components` bo nie jest komponentem. 
Jak już to `src`.